### PR TITLE
fix: eslint config

### PR DESCRIPTION
**Pull request by krishguptadev**
resolves #`issueNumber`

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes

- Eslint
- Fixes eslint issues

## Notes

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/awesome-github-profiles/pull/926"></a>

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
I think it doesnt need to be that complicated, next offers eslint with a prettier integration without much configuration effort : [See here](https://nextjs.org/docs/basic-features/eslint)

**Comment by krishguptadev** [comment]
> I think it doesnt need to be that complicated, next offers eslint with a prettier integration without much configuration effort : [See here](https://nextjs.org/docs/basic-features/eslint)

Actually, this PR is using the next feature but uses the stylistic rules from all of our other repos (LinkFree, EventCalendar)

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+env:
+  browser: true
+  es2021: true
+extends:
+  - 'plugin:react/recommended'
```
Comment (Cahllagerfeld) [inline .eslintrc.yml:5]: > I think it doesnt need to be that complicated, next offers eslint with a prettier integration without much configuration effort : [See here](https://nextjs.org/docs/basic-features/eslint)

This uses the `eslint`-configuration from the Linkfree repo (which is a cra-app) mixed with some next-configuration right?

**Review by Cahllagerfeld — COMMENTED** [review]
As mentioned in my comment above, these changes aren't necessary in my opinion. We can adjust the eslint-config no question, but imo we shouldn't mix configs from a `create-react-app` and a `next` app. What do you think about this @eddiejaoude

File: .eslintrc.yml (hunk 1 above)
Comment (krishguptadev) [reply .eslintrc.yml:5]: Yes! But we still need the react plugin right? Because we are working with JSX?

**Comment by eddiejaoude** [comment]
Thanks for the PR @krishguptadev - I think we should stick to vanilla NextJS config, otherwise it will be confusing - I really like to keep things as the framework recommends with minimal customisation as possible.

Great collaboration everyone 💪

**Pull request closed without merging**